src/App.css

```diff
@@ -0,0 +1,57 @@
+html,
+body {
+  font-size: 12px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.App {
+  margin: 1rem;
+}
+
+.container {
+  width: 100%;
+}
+
+.center-container {
```

Comment:
.scss/.css files I usually see as lower cased not upper cased

Reply:
Yeah, I agree. For some reason I've seen react apps using capitalized filenames for .css stuff, and I'm not exactly sure why / when that's helpful. My guess is that it's supposed to be .css that's only pertinent to the corresponding component. If that's the case, I should move those css rules elsewhere.

Reply:
Yep, I've often seen this employed in a component-centric architecture. Thing.jsx, Thing.css, Thing.test.jsx, Thing.story.jsx, etc.